src/PLEXOSSolutionDataset.jl

```diff
@@ -31,7 +31,8 @@ function PLEXOSSolutionDataset(
     return PLEXOSSolutionDataset(xml)
 end
 
-function PLEXOSSolutionDataset(xml::Document)
+function PLEXOSSolutionDataset(xml::Document;
+    ignored_tables::Vector{String}=IGNORED_TABLES)
 
     summary = PLEXOSSolutionDatasetSummary(xml)
```

Comment:
If `ignored_tables` is passed in as an argument, we would want to propagate that through here as well, no?

It may be clearer to stick to one way to provide the list of tables - either as arguments or by setting the global variable, but not a mix of the two. I'm leaning slightly towards the latter, to avoid having to propagate the arguments down through any calling code (e.g. H5PLEXOS.jl). I suppose the argument against that might be if you're processing multiple files with different versions and need to exclude different tables in each, and so need to change the global variable each time. I'm not sure how likely that is though.

Reply:
Thanks, I agree that we should propagate as `PLEXOSSolutionDatasetSummary(xml; ignored_tables)` as well here.

Reply:
I've been thinking over how `plexostables` and `plexostables_lookup` are treated as a globals too.